<ui version="4.0" >
 <class>isoContourDialog</class>
 <widget class="QDialog" name="isoContourDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>260</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Apply</string>
   </property>
   <property name="icon" >
    <iconset resource="../ElmerGUI.qrc" >:/icons/dialog-ok-apply.png</iconset>
   </property>
   <property name="default" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>260</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="icon" >
    <iconset resource="../ElmerGUI.qrc" >:/icons/dialog-close.png</iconset>
   </property>
  </widget>
  <widget class="QComboBox" name="contoursCombo" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>20</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="colorCombo" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>150</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="variableLabel" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>20</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Variable:</string>
   </property>
  </widget>
  <widget class="QLabel" name="colorLabel" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contoursMinEdit" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="contoursMaxEdit" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="contoursSpin" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum" >
    <number>1</number>
   </property>
   <property name="maximum" >
    <number>99999</number>
   </property>
   <property name="value" >
    <number>10</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="colorMinEdit" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="colorMaxEdit" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="contoursLabel" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Contours:</string>
   </property>
  </widget>
  <widget class="QLabel" name="contoursMinLabel" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>80</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="contoursMaxLabel" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>110</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Max:</string>
   </property>
  </widget>
  <widget class="QLabel" name="colorMinLabel" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>180</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="colorMaxLabel" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>210</y>
     <width>31</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Max:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc" />
 </resources>
 <connections/>
</ui>
